CHANGELOG.md

```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.2.0]
```

Comment:
Missing a `[1.2.0]: ...` link associated with this reference link.
Looks like everything after 1.0.0 is missing it.
This results in this: https://github.com/uber-go/goleak/blob/b3e1892c073e7684a5b17fc928ab6751e79a7550/CHANGELOG.md
(Also looks like we accidentally tagged 1.1.10 after 1.1.0 but that's not an issue with 1.2.0 anymore.)

Reply:
yeah, I was wondering why 1.1.10 happened after 1.1.0 but 🤷 Thanks for making the changes here.